index.bs

```diff
@@ -705,12 +756,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     up a [=client platform=].
 
 : <dfn>[WRP]</dfn>
-:: The entity whose web application utilizes the [=Web Authentication API=] to register and authenticate users. See
-    [=Registration=] and [=Authentication=], respectively.
+:: The entity whose <dfn>web application</dfn> utilizes the [[#api|Web Authentication API]] to [=registration|register=] and
+    [=authentication|authenticate=] users.
 
-        Note: While the term [=[RP]=] is used in other contexts (e.g., X.509 and OAuth), an entity acting as a [=[RP]=] in one
+        Note: While the term [=[RP]=] is also often used in other contexts (e.g., X.509 and OAuth), an entity acting as a [=[RP]=] in one
         context is not necessarily a [=[RP]=] in other contexts. In this specification, the term [=[WRP]=] is often shortened 
-        to be just [=[RP]=].
+        to be just [=[RP]=], and explicitly refers to a [=[RP]=] in the WebAuthn context. Note that in any concrete instantiation
+        that a WebAuthn context may be embedded in a broader overall context, e.g., one based on OAuth.
```

Comment:
I can't quite parse the last sentence - I think there might be one too many "that"s in it, or something similar.

Reply:
ok, 2nd "that" removed. better?